src/lib/components/SideNav/index.js

```diff
@@ -58,7 +58,11 @@ class SideNav extends BaseElement {
       }
     }
     return html`
-      <nav @click="${this.onBlockClicks}" class="web-side-nav__container">
+      <nav
```

Comment:
Unfortunately these changes break keyboard support for the responsive side nav. I think you can safely revert them.